jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebAppContext.java

```diff
@@ -987,6 +988,8 @@ else if (getBaseResource() != null)
         name = String.format("%s@%x", name, hashCode());
 
         dumpObjects(out, indent,
+            Dumpable.named("maxFormKeys ", getMaxFormKeys()),
```

Comment:
Rather than override dump and have to repeat the stuff from the base class, why don't we just add all these Dumpable collections as beans the the context and let the normal dump mechanism dump them?

Reply:
Or better yet, add one Dumpable webapp bean, that dumps all these extra details

Reply:
I opened a separate issue for this (see https://github.com/jetty/jetty.project/issues/12655). 
As its outside the scope of this documentation PR.